### DISCUSSION TITLE
Is there any way to publish the dendron knowledge graph?

### DISCUSSION THREAD

**Discussion by thanhpp** [dendronhq/dendron#3397] [Q&A] [answered]
[Obsidian Publish has an integrated knowledge graph in the published website](https://knowledge.grokking.org/Topics/Artificial+Intelligence). Can dendron do it? Maybe just the image only

**Comment by rlh1994** [comment, marked as answer]
Currently this isn't possible, but there is an open feature request for it that you can thumbs up to encourage it to be implemented #1945

**Reply by thanhpp** [reply]
it's nice to know. Thanks a lot.
